addons/insurgents/groups/blufor.hpp

```diff
@@ -0,0 +1,76 @@
+class West {
+    class Guerilla {
+        class Infantry {
+            class GVAR(BLU_G_F_sentry) {
+                faction = "BLU_G_F";
+                name = CSTRING(Group_Sentry);
+                icon = "\A3\ui_f\data\map\markers\nato\b_inf.paa";
+                side = 1;
+
+                class Unit0 {
+                    position[] = {0, 0, 0};
+                    rank = "CORPORAL";
+                    side = 1;
+                    vehicle = QGVAR(soldier_b);
+                };
+                class Unit1: Unit0 {
+                    position[] = {5, -5, 0};
+                    rank = "PRIVATE";
+                };
+            };
+            class GVAR(BLU_G_F_fireteam): GVAR(BLU_G_F_sentry) {
+                name = CSTRING(Group_FireTeam);
+
+                class Unit0 {
+                    position[] = {0, 0, 0};
+                    rank = "SERGEANT";
+                    side = 1;
+                    vehicle = QGVAR(soldier_b);
+                };
+                class Unit1: Unit0 {
+                    position[] = {5, -5, 0};
+                    rank = "PRIVATE";
+                };
+                class Unit2: Unit1 {
```

Comment:
All subsequent units should inherit from `Unit0` to avoid unnecessary nested inheritance trees.

Reply:
Didn't do it as you said.

Subsequent units inherit from the last one with `rank = "PRIVATE";` to preserve command structures.

Reply:
Yeah that's fine. What I meant is you shouldn't do inheritance `Unit0 -> Unit1 -> Unit2 -> Unit3 -> Unit4` if you can just do `Unit0 -> Unit1/2/3/4` instead.